src/Parser/JmesPathParser.cs

```diff
@@ -0,0 +1,291 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using StarodubOleg.GPPG.Runtime;
+using DevLab.JmesPath.Tokens;
+using DevLab.JmesPath.Utils;
+
+namespace DevLab.JmesPath
+{
+    public interface IJmesPathGenerator
+    {
+        void OnExpression();
```

Comment:
Addressed with 1a03730f36469f4eb8391b912d6a3e06c6d8ee40, although instead of sorting alphabetically (which doesn't seem to improve things), I've tried to group them. Let me know if that's the effect you were after.